### PR TITLE
rename most resin to balena

### PR DESCRIPTION
**Pull request by xginn8**
Connects-to: https://github.com/balena-os/resinhup/issues/266
Signed-off-by: Matthew McGinn <matthew@balena.io>
Change-type: patch

### CODE REVIEW

**Comment by xginn8** [comment]
fwiw i'm ignoring the other two upgrade scripts (1.x and 1.x->2.x) as they are on their way into the sunset and i don't want to take any risks preventing upgrades.

File: upgrade-2.x.sh [hunk 1]
```diff
@@ -1026,19 +1026,19 @@ if version_gt "$VERSION_ID" "2.0.6" &&
...
 
-# Fix for issue: https://github.com/resin-os/meta-resin/pull/864
-# Also includes change from: https://github.com/resin-os/meta-resin/pull/882
+# Fix for issue: https://github.com/balena-os/meta-balena/pull/864
+# Also includes change from: https://github.com/balena-os/meta-balena/pull/882
 if version_gt "$VERSION_ID" "2.0.7" &&
     version_gt "2.7.0" "$VERSION_ID"; then
         log "Fixing supervisor updater..."
-        if CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 --fail --silent -o "/tmp/update-resin-supervisor" https://raw.githubusercontent.com/resin-os/meta-resin/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor ; then
+        if CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 --fail --silent -o "/tmp/update-resin-supervisor" https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor ; then
```
Comment (saintaardvark) [inline upgrade-2.x.sh:1041]: As before, this gives me a 404; is this expected?

Link: https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

Try to download with curl:
```
curl -i https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor
HTTP/1.1 404 Not Found
Connection: keep-alive
Content-Length: 14
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
Content-Type: text/plain; charset=utf-8
Via: 1.1 varnish (Varnish/6.0)
X-GitHub-Request-Id: 89EE:2BE1:11934:1712D:5EBEF4A2
Accept-Ranges: bytes
Date: Fri, 15 May 2020 19:59:30 GMT
Via: 1.1 varnish
X-Served-By: cache-sea4476-SEA
X-Cache: HIT, MISS
X-Cache-Hits: 1, 0
X-Timer: S1589572771.712788,VS0,VE7
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
X-Fastly-Request-ID: 90ad47b6a4fe18bc28682a0dfcab157f378aa58a
Expires: Fri, 15 May 2020 20:04:30 GMT
Source-Age: 0

404: Not Found
```

Comment (xginn8) [reply upgrade-2.x.sh:1041]: Good catch! updated.

File: upgrade-2.x.sh [hunk 2]
```diff
@@ -35,7 +35,7 @@ fi
 ###
 
 # Preventing running multiple instances of upgrades running
-LOCKFILE="/var/lock/resinhup.lock"
+LOCKFILE="/var/lock/balenahup.lock"
```
Comment (xginn8) [inline upgrade-2.x.sh:38]: It may be worth leaving this in place to not race against long-running HUPs...

File: upgrade-2.x.sh [hunk 3]
```diff
@@ -787,8 +787,12 @@ fi
 # Log timer
 starttime=$(date +%s)
 
+# For compatibility purposes
```
Comment (xginn8) [inline upgrade-2.x.sh:790]: @saintaardvark leaving this around in case we rely on this somehow (or customers do)

File: upgrade-2.x.sh (hunk 2 above)
Comment (xginn8) [reply upgrade-2.x.sh]: I reverted this change and left the old path in place.

**Pull request merged**